processors/record_linker/processor.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import uuid
+import logging
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+# Module API
+
+def process(conf, conn):
+
+    # Trial m2m
+    TRIAL_TABLES = [
```

Comment:
I think better to get an error on PaperTrials if there is a documents related to bad trial. So then we could decide what to do with it.